libs/martech/helpers.js

```diff
@@ -1,63 +1,42 @@
+/* eslint-disable no-underscore-dangle */
 const AMCV_COOKIE = 'AMCV_9E1005A551ED61CA0A490D45@AdobeOrg';
 const KNDCTR_COOKIE_KEYS = [
   'kndctr_9E1005A551ED61CA0A490D45_AdobeOrg_identity',
   'kndctr_9E1005A551ED61CA0A490D45_AdobeOrg_cluster',
 ];
 
+let DATA_STREAM_ID = '';
+
 function getDomainWithoutWWW() {
   const domain = window?.location?.hostname;
   return domain.replace(/^www\./, '');
 }
 
-/**
```

Comment:
I agree that we should trim or remove these comments.  We may want to keep some of this info though - could create a helpers.md file with some of these details?

At the very least mention that comments are being removed in the PR description so that devs can always look back in history.